src/main/java/com/ibm/devops/connect/DevOpsGlobalConfiguration.java

```diff
@@ -72,6 +74,15 @@ public void setSyncToken(String syncToken) {
         save();
     }
 
+    public String getApiToken() {
+        return apiToken;
+    }
+
+    public void setApiToken(String apiToken) {
+        this.apiToken = apiToken;
```

Comment:
apiToken in this case is user access key?

Reply:
Yes.  Hesitating to change it as it would break people's config page if we change.  The label is proper in the UI

Reply:
Could we at least get a comment that states this?